packages/next/src/server-build.ts

```diff
@@ -2101,11 +2101,29 @@ export async function serverBuild({
               ),
               check: true,
             },
+            {
+              src: path.posix.join(
+                '/',
+                entryDirectory,
+                '(.+)/\\.prefetch\\.rsc$'
+              ),
+              dest: path.posix.join(
+                '/',
+                entryDirectory,
+                `$1${RSC_PREFETCH_SUFFIX}`
+              ),
+              check: true,
+            },
             {
               src: path.posix.join('/', entryDirectory, '/\\.rsc$'),
               dest: path.posix.join('/', entryDirectory, `/index.rsc`),
               check: true,
             },
+            {
+              src: path.posix.join('/', entryDirectory, '(.+)/\\.rsc$'),
```

Comment:
```suggestion
              src: path.posix.join('/', entryDirectory, '/(.+)\\.rsc$'),
```
Should the group match be after the `/`?